samples/python/onedim/diffusion_flame_continuation.py

```diff
@@ -54,43 +62,139 @@
 # Flame Continuation
 # ------------------
 
+# Maximum number of steps to take
+Nmax = 1000
```

Comment:
```suggestion
 n_max = 1000
```
With the exception of uppercase T, P, X, etc for states, I’d prefer to stick to Python default snake_case.